src/interface/src/app/treatments/prescription-actions/prescription-actions.component.html

```diff
@@ -0,0 +1,5 @@
+<div *ngIf="prescriptions$ | async; let prescriptions" class="prescriptions">
+  <button *ngFor="let action of prescriptions.SINGLE | keyvalue" [value]="">
```

Comment:
did you mean to leave value blank here?

Reply:
yeah that's a placeholder, im not doing assignments on this PR, to limit the scope of changes.